### DISCUSSION TITLE
Impossible to install Aquacrop-ospy in Pycharm. HELP!

### DISCUSSION THREAD

**Discussion by paolaguzmanluna** [aquacropos/aquacrop#101] [General]
I cannot install Aquacrop-Ospy in Pycharm. I am trying to do it throught the pycharm console but I get an error after running: pip install aquacrop

Does anybody have the same problem?

**Comment by Paloschi** [comment]
Hello Paola, could you please share the error message?

**Comment by ZhiyiLiao** [comment]
芷依已收到您发送的邮件，谢谢！

**Comment by Paloschi** [comment]
If the error is related to the Missing Compiler:
The RuntimeError regarding the absence of a suitable MSVC (Microsoft Visual C++) compiler is more critical. AquaCrop-OS's use of Numba for AOT compilation requires a C compiler to be available:

For Windows: Ensure that you have the Microsoft C++ Build Tools installed. You can download them from the Visual Studio downloads page (look for "Build Tools for Visual Studio"). After installation, you might need to restart your command prompt or IDE to ensure the environment variables are updated.

For Linux/Mac: Install GCC (GNU Compiler Collection) if it's not already present. This can usually be done through your package manager (apt for Ubuntu, brew for macOS, etc.).

**Reply by florentdemelezi** [reply]
But even after installing the required msvc does not work? Is there a specific version which need to be implemented
